{
  "schema_version": "1.2.0",
  "id": "GHSA-jg4f-jqm5-4mgq",
  "modified": "2021-09-14T19:40:45Z",
  "published": "2018-10-10T17:23:14Z",
  "aliases": [
    "CVE-2016-8628"
  ],
  "summary": "Moderate severity vulnerability that affects ansible",
  "details": "Ansible before version 2.2.0 fails to properly sanitize fact variables sent from the Ansible controller. An attacker with the ability to create special variables on the controller could execute arbitrary commands on Ansible clients as the user Ansible runs as.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.0/AV:N/AC:L/PR:H/UI:N/S:C/C:H/I:H/A:H"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "ansible"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2.2.0"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2016-8628"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-jg4f-jqm5-4mgq"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/ansible/ansible"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-77"
    ],
    "severity": "CRITICAL",
    "github_reviewed": true
  }
}